components/core/marketing/Issue.js

```diff
@@ -0,0 +1,119 @@
+import React, { Component } from "react";
```

Comment:
Pretty much every file in the codebase does `import * as React from "react"`; so lets keep it all uniform.